### DISCUSSION TITLE
measure descriptions

### DISCUSSION THREAD

**Discussion by limajo** [sql-bi/Bravo#606] [Ideas]
any plans to add measure descriptions to the time intelligent dax functions created by bravo?

**Comment by albertospelta** [comment]
Bravo does not provide a default description for the created DAX measures but you can customize this behavior. To add a description to a DAX measure, you need to create a custom template and insert the `Description` property in the `TimeIntelligence.json` file, as shown below.

![image](https://user-images.githubusercontent.com/1957788/222145983-da32b14d-21e5-4b6b-b0c7-605316653a5d.png)

You can find more details on how to customize templates here https://docs.sqlbi.com/bravo/features/manage-dates/customize-date-template
